### PR TITLE
[fr] webextensions/chrome_incompatibilities: Small missing translation fix

### PR DESCRIPTION
**Pull request by busybox11**
### Description

Edited a word that wasn't translated.

### Motivation

Felt weird when reading the docs? It's small but nice to have I guess.

### Additional details

/

### CODE REVIEW

File: files/fr/mozilla/add-ons/webextensions/chrome_incompatibilities/index.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Le reste de cette section décrit les problèmes de compatibilité qui ne sont p
...
-- Dans Firefox, à partir de Firefox 52, certaines API `browser.webRequest.*` renvoient des promesses qui résolvent `webRequest.BlockingResponse` de façon asynchrone. Pour Chrome, seule `webRequest.onAuthRequired` prend en charge une gestion asynchrone de `webRequest.BlockingResponse` avec `'asyncBlocking'`.
+- Dans Firefox, à partir de Firefox 52, certaines API `browser.webRequest.*` renvoient des promesses qui résolvent `webRequest.BlockingResponse` de façon asynchrone. Pour Chrome, seule `webRequest.onAuthRequired` prend en charge une gestion asynchrone de `webRequest.BlockgResponse` avec `'asyncBlockg'`.
```
Comment (cw118) [inline files/fr/mozilla/add-ons/webextensions/chrome_incompatibilities/index.md:104]: I believe the previous text is correct (cf. [the en-US counterpart](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities#webrequest_api)).

```suggestion
- Dans Firefox, à partir de Firefox 52, certaines API `browser.webRequest.*` renvoient des promesses qui résolvent `webRequest.BlockingResponse` de façon asynchrone. Pour Chrome, seule `webRequest.onAuthRequired` prend en charge une gestion asynchrone de `webRequest.BlockingResponse` avec `'asyncBlocking'`.
```

File: files/fr/mozilla/add-ons/webextensions/chrome_incompatibilities/index.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ Le reste de cette section décrit les problèmes de compatibilité qui ne sont p
 
 #### declarativeContent
 
-l'API `declarativeContent` de Chrome [n'a pas encore été implémentée](https://bugzilla.mozilla.org/show_bug.cgi?id=1435864) in Firefox.
+l'API `declarativeContent` de Chrome [n'a pas encore été implémentée](https://bugzilla.mozilla.org/show_bug.cgi?id=1435864) dans Firefox.
```
Comment (cw118) [inline files/fr/mozilla/add-ons/webextensions/chrome_incompatibilities/index.md:114]: I take advantage of the PR to fix another small typo :)

```suggestion
L'API `declarativeContent` de Chrome [n'a pas encore été implémentée](https://bugzilla.mozilla.org/show_bug.cgi?id=1435864) dans Firefox.
```

**Review by cw118 — CHANGES_REQUESTED** [review]
Thank you for the pull request @busybox11 ! I added some small comments :)

**Comment by busybox11** [comment]
Aha, nice catch @cw118, thanks for the heads up :) I committed your changes. Thank you so much!

**Review by cw118 — APPROVED** [review]
This is great, thank you so much for this contribution!

**Pull request merged**